docs/UsersGuide/source/CustomizingTheWorkflow/ConfigWorkflow.rst

```diff
@@ -331,47 +566,132 @@ Grid Generation Parameters
 
    If the experiment uses a **user-defined grid** (i.e., if ``PREDEF_GRID_NAME`` is set to a null string), then ``GRID_GEN_METHOD`` must be set in the experiment configuration file. Otherwise, the experiment generation will fail because the generation scripts check to ensure that the grid name is set to a non-empty string before creating the experiment directory.
 
+.. _PredefGrid:
+
+Predefined Grid Parameters
+------------------------------
+
+``PREDEF_GRID_NAME``: (Default: "")
+   This parameter indicates which (if any) predefined regional grid to use for the experiment. Setting ``PREDEF_GRID_NAME`` provides a convenient method of specifying a commonly used set of grid-dependent parameters. The predefined grid settings can be viewed in the script ``ush/set_predef_grid_params.yaml``. 
+   
+   **Currently supported options:**
+   
+   | ``"RRFS_CONUS_25km"``
+   | ``"RRFS_CONUS_13km"``
+   | ``"RRFS_CONUS_3km"``
+   | ``"SUBCONUS_Ind_3km"`` 
+   
+   **Other valid values include:**
+
+   | ``"AQM_NA_13km"``
+   | ``"CONUS_25km_GFDLgrid"`` 
+   | ``"CONUS_3km_GFDLgrid"``
+   | ``"EMC_AK"`` 
+   | ``"EMC_HI"`` 
+   | ``"EMC_PR"`` 
+   | ``"EMC_GU"`` 
+   | ``"GSL_HAFSV0.A_25km"`` 
+   | ``"GSL_HAFSV0.A_13km"`` 
+   | ``"GSL_HAFSV0.A_3km"`` 
+   | ``"GSD_HRRR_AK_50km"``
```

Comment:
These grids have been removed as part of PR #871:

- EMC_AK
- EMC_HI
- EMC_PR
- EMC_GU
- GSL_HAFSV0.A_25km
- GSL_HAFSV0.A_13km
- GSL_HAFSV0.A_3km
- GSD_HRRR_AK_50km

Reply:
Removed